samples/bluetooth/mesh/ble_peripheral_lbs_coex/sample.yaml

```diff
@@ -9,4 +9,6 @@ tests:
     integration_platforms:
       - nrf52dk_nrf52832
       - nrf52840dk_nrf52840
+      - nrf52833dk_nrf52833
+      - nrf52833dk_nrf52820
```

Comment:
The nrf52820 should be removed from documentation instead of adding it here. We will not claim support for the nrf52820 because it does not have enought RAM. This is for all the samples. We do not have a QDID for nRF52820.

Reply:
So then the questions comes if we rather should add the nRF5340? Both here and in doc.

Reply:
Reference to nRF52820 removed.
Also removed _ns build targets.